libraries/Microsoft.Bot.Builder.Dialogs/SkillDialog.cs

```diff
@@ -91,15 +91,6 @@ public override async Task<DialogTurnResult> BeginDialogAsync(DialogContext dc,
         /// return value.</remarks>
         public override async Task<DialogTurnResult> ContinueDialogAsync(DialogContext dc, CancellationToken cancellationToken = default)
         {
```

Comment:
If I remember correctly, this logic was added to allow a skill to resume after an interruption, we should test this scenario if we decide to keep events.